FEXCore/unittests/Emitter/System_Tests.cpp

```diff
@@ -6,6 +6,9 @@
 
 using namespace ARMEmitter;
 
+TEST_CASE_METHOD(TestDisassembler, "Emitter: Reserved") {
+  TEST_SINGLE(udf(0), "udf #0x0");
```

Comment:
Does it make sense to add a test case for the maximum reserved value as well?

Reply:
Sure, added one.